Automattic/jetpack
# New Jumpstart UI!

Takes care of #7373 

To Test: 
- Make sure Jetpack is connected
- Click `reset options` in the footer
- You should see Jumpstart 
- Try Jumpstarting
- Try x-ing out of the Jumpstart 

Should look like this:
![image](https://user-images.githubusercontent.com/1123119/28291560-d1982860-6aff-11e7-8473-edbee5e8ef50.png)
